Solutions/Azure Storage/Package/mainTemplate.json

```diff
@@ -35,9 +35,9 @@
     "email": "support@microsoft.com",
     "_email": "[variables('email')]",
     "workspaceResourceId": "[resourceId('microsoft.OperationalInsights/Workspaces', parameters('workspace'))]",
-    "uiConfigId1": "AzureStorageAccount_CCP",
+    "uiConfigId1": "AzureStorageAccount",
```

Comment:
@v-sabiraj : this would be generating from the id property of the data connector... can you please check and update that as well, so that this issue does not happen again.... Thanks!!!